odoo/fields.py

```diff
@@ -335,6 +335,7 @@ class Second(models.Model):
 
         'related_field': None,          # corresponding related field
         'group_operator': None,         # operator for aggregating values
+        'group_full': None,             # name of method to retrieve all groups
```

Comment:
Not fond of "group full" as an attribute name, it doesn't necessarily return "full" anything after all (I'm not sure it even has to return *more* than it got)